scripts/import_assets/README.md

```diff
@@ -0,0 +1,46 @@
+# Importing assets
+
+
+## Description 
+Post request for assets creation by attaching CSV files stored locally. 
+## prerequisites
+* python3
+
+## Inputs arguments
+To use this code, the following inputs are requierd; 
+* CSV file containg the assets in the following format:
+
+```
+name, type, business impact, details, tags, hosts personal data, ip/ip range
+```
+For instance: 
+
+
+```python
```

Comment:
Please remove "python" from here. This block is CSV, so it doesn't need "python" marker.

Reply:
Done!